sdk/keyvault/keyvault-admin/review/keyvault-admin.api.md

```diff
@@ -19,10 +19,28 @@ export interface AccessControlClientOptions extends CommonClientOptions {
     serviceVersion?: SUPPORTED_API_VERSIONS;
 }
 
+// @public
+export interface BooleanKeyVaultSetting {
+    name: string;
+    type: KnownSettingType.Boolean;
+    value: boolean;
+}
+
+// @public
+export interface CreateOrUpdateSettingOptions extends OperationOptions {
```

Comment:
We no longer have a CreateOrUpdate operation. It's just GetSetting, UpdateSetting, and ListSettings. One can't create settings, it turns out. The wrong assumption was made based on the pattern they used in swagger.